website/static/js/pages/project-dashboard-page.js

```diff
@@ -201,7 +200,16 @@ $(document).ready(function () {
                 }
             };
             var filebrowser = new Fangorn(fangornOpts);
+            if (newComponentElem) {
+                m.mount(newComponentElem, AddComponentButton);
+            };
+            m.endComputation();
         });
+    } else {
```

Comment:
This else (and the mount/endcomp after) isn't needed as it only occurs when the node is retracted, in which case you don't need to have an addComponent button.